.github/workflows/end-to-end-tests.yml

```diff
@@ -0,0 +1,100 @@
+name: End-to-End Tests
+
+on:
+  push:
+    branches: [trunk]
+  pull_request:
+    branches: [trunk]
+
+jobs:
+  E2E_Tests:
+    name: End to end tests
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Cache node modules
```

Comment:
Nice! I'm assuming this will cache `node_modules` to speed up individual jobs. Should we cache `composer` deps too?

Reply:
It doesn't cache `node_modules` but rather the `npm` cache. This, I believe keeps all the packages locally, so when `npm ci` runs, rather than fetching from npm it just pulls them from the cache.

Good idea about composer too, I added that to the cache.